exportify.js

```diff
@@ -210,9 +210,13 @@ let PlaylistExporter = {
 					if (song.track?.album && song.track.album.id) { album_ids.add(song.track.album.id) }
 					// Multiple, comma-separated artists can throw off csv, so surround with "". Same for track and album names,
```

Comment:
If we're no longer comma-separating artists, say we're semicolon-separating them. Code comments need to agree with the code.